config/rbac/user-role.yaml

```diff
@@ -0,0 +1,54 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  creationTimestamp: null
+  name: user-role
```

Comment:
Name is not indicative, also false since this is a `ClusterRole` and not a `Role`